plugin/efficientNMSPlugin/efficientNMSPlugin.cpp

```diff
@@ -493,6 +494,10 @@ IPluginV2DynamicExt* EfficientNMSPluginCreator::createPlugin(const char* name, c
                 PLUGIN_VALIDATE(scoreSigmoid == 0 || scoreSigmoid == 1);
                 mParam.scoreSigmoid = static_cast<bool>(scoreSigmoid);
             }
+            if (!strcmp(attrName, "class_agnostic"))
+            {
+                mParam.classAgnostic = *(static_cast<const bool*>(fields[i].data));
```

Comment:
East-const is preferred.
```suggestion
                mParam.classAgnostic = *(static_cast<bool const*>(fields[i].data));
```